tm2/pkg/commands/ffcli/command.go

```diff
@@ -116,6 +116,19 @@ func (c *Command) Parse(args []string) error {
 				return subcommand.Parse(c.args[1:])
 			}
 		}
+		// args[0] is not a sub command, so it's an command argument.
+		c.args = c.args[:1]
+		// continue flag parsing after args[0]
+		for c.FlagSet.NArg() > 1 {
+			err := ff.Parse(c.FlagSet, c.FlagSet.Args()[1:], c.Options...)
```

Comment:
Do we already support the famous `—`?

Reply:
Good point, the `--` is ignored when the flags are after the arguments, but not before, as illustrated in the examples below.

I will add these scenarios to the test cases and see how to fix that.

With this PR version : 
```sh
$ gnokey query -remote https://test3.gno.land:36657 \ 
    auth/accounts/g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5
// Pass

$ gnokey query -- -remote https://test3.gno.land:36657 \ 
    auth/accounts/g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5  
// Fail because query expect a single arg

$ gnokey query -- auth/accounts/g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5  \
    -remote https://test3.gno.land:36657
// Pass but it shouldn't because of `--`
```

With `master` version : 
```sh
$ gnokey query -remote https://test3.gno.land:36657 \
    auth/accounts/g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5  
// Pass


$ gnokey query -- -remote https://test3.gno.land:36657 \ 
    auth/accounts/g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5  
// Fail because query expect a single arg

$ gnokey query -- auth/accounts/g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5  \
    -remote https://test3.gno.land:36657
// Fail because query expect a single arg
```